sdk/translation/ai-translation-text-rest/src/index.ts

```diff
@@ -1,16 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import TextTranslationClient from "./custom/customClient";
+import TextTranslationClient from "./customClient";
```

Comment:
I think custom client is renamed ? Can we have a way to keep the original client in generated folder and have the customize client call generated client?

Reply:
If I understand your ask correctly, you want me not to remove the generated client, keep the file in there and just reference the custom one here?

Reply:
yes, exactly,  I asked this way because generated folder could be override by the codegen when we refresh the SDK next time. it will be disturbing if we change anything inside there.

Reply:
Done